infra/terraform/test-org/org/locals.tf

```diff
@@ -849,6 +849,13 @@ locals {
       owners      = ["prabhu34"]
       topics      = join(",", [local.common_topics.containers, local.common_topics.devtools])
     },
+    {
+      name        = "terraform-google-bigtable"
+      org         = "GoogleCloudPlatform"
```

Comment:
@hariprabhaam I don't see this repo in GoogleCloudPlatform. Has this been published?

Reply:
Yes it is published @bharathkkb  https://github.com/GoogleCloudPlatform/terraform-google-bigtable